docs/GetStarted/getting_started_linux_cmake.md

```diff
@@ -16,36 +16,40 @@ documented separately, as they require further setup.
 
 ## Prerequisites
 
-### Install CMake
+### Generic and IDE setup
```

Comment:
Can you also update the section title + subheader levels in the Windows docs? https://github.com/google/iree/blob/master/docs/GetStarted/getting_started_windows_cmake.md